Code/JavaWrappers/ChemReactions.i

```diff
@@ -72,6 +73,17 @@ static std::string ReactionToSmarts(ChemicalReaction &rxn) {
 static std::string ReactionToRxnBlock (const ChemicalReaction &rxn) {
   return RDKit::ChemicalReactionToRxnBlock(rxn);
 };
+/*
+static RDKit::ROMol *ReduceProductToSideChains(RDKit::ROMOL_SPTR product,
+                                               bool addDummyAtoms=true) {
+  return RDKit::reduceProductToSideChains(product, addDummyAtoms);
+};
+*/
+static RDKit::ROMol *ReduceProductToSideChains(RDKit::ROMol *product,
```

Comment:
Yeah, they may well need it. There are a pile of other functions that return new objects that would likely benefit. 

That's for the next time I dig into the SWIG wrappers I guess.